modules/prepare_annotation.nf

```diff
@@ -5,34 +5,36 @@ process format_annotation {
     label 'python3'
     label 'smallTask'
 
-    if ( params.softlink_results ) { publishDir "${params.output}/${params.annotation_dir}", pattern: "*.id2ensembl" }
-    else { publishDir "${params.output}/${params.annotation_dir}", mode: 'copy', pattern: "*.id2ensembl" }
+    if ( params.softlink_results ) { publishDir "${params.output}/${params.annotation_dir}", pattern: "*.id2details" }
```

Comment:
good idea to generalize this file name! But should we also change the variable in the deseq2.R script from 

`ensembl2genes` to `id2name` or something like this?

Reply:
yes, that sounds reasonable